CMakeLists.txt

```diff
@@ -208,7 +208,7 @@ if(SINGULARITY_USE_KOKKOS)
   # type: soft submodule 
   singularity_import_dependency(
     PKG Kokkos
-    TARGET Kokkos::kokkos
+    TARGETS Kokkos::kokkos
```

Comment:
what is the difference between these?

Reply:
An `S`

the macro took as input `TARGET` and `TARGETS`, These did the same thing, but just bifurcated into singular and plural. I changed this to only one input `TARGETS`